patches/third_party-blink-renderer-core-html-html_html_element.cc.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/third_party/blink/renderer/core/html/html_html_element.cc b/third_party/blink/renderer/core/html/html_html_element.cc
+index 7cc4857d439f055e3e3ed24d0f8ebd29a9a93fb9..082be842010cf84110d5b310490110f5b212a237 100644
+--- a/third_party/blink/renderer/core/html/html_html_element.cc
++++ b/third_party/blink/renderer/core/html/html_html_element.cc
+@@ -64,6 +64,36 @@ void HTMLHtmlElement::InsertedByParser() {
+   }
+ }
+ 
++void HTMLHtmlElement::AttributeChanged(const AttributeModificationParams& params) {
++  // Check to see if the following conditions are true, to work around
```

Comment:
This is a large patch. Is there a way we can patch in a hook to a more generic `AttributeChanged` brave function, either in chromium_src, or brave?

Reply:
@petemill sure, this could be revised to pass a `AttributeModificationParams` into some func (`PossiblyModifyAttributeModificationParamsToIgnoreAsyncHide`?) which could pass back the same, or a modified `AttributeModificationParams` as necessary.

I dont have much experience with that pattern at Brave though (all my work has been 100% in blink).  Who would be good to ask for some handholding there :)

Reply:
I think the simplest thing we could do here is to patch in a generic function: `PossiblyModifyAttributeModificationParams`. And then define that function in `brave/chromium_src/third_party/blink/renderer/core/html/html_html_element.cc`. That file could look something like:

```cpp
// define function for `PossiblyModifyAttriguteModificationParams`
AttributeModificationParams PossiblyModifyAttriguteModificationParams(const AttributeModificationParams& original_params) {
  // ...decide to do something or not...
}

// include original chromium blink source which is patched to call above function
#include "../../../../../..//third_party/blink/renderer/core/html/html_html_element.cc"
```

What this buys us is that the patch is minimized, the code is in a .cc file and not a .patch file, and when we want to add more modifications, we might not need to do any further patching. Seems like a minimal improvement, but I think that's the pattern we're going for right now.